ably-ios/ARTTypes.h

```diff
@@ -47,9 +47,7 @@ typedef NS_ENUM(NSUInteger, ARTRealtimeChannelState) {
     ARTRealtimeChannelInitialised,
     ARTRealtimeChannelAttaching,
     ARTRealtimeChannelAttached,
-    ARTRealtimeChannelDetaching,
```

Comment:
As mentioned before, the only definitive spec is at http://docs.ably.io/client-lib-development-guide/features/.  I am working on bringing all the other docs up to date, but they're not there yet.